.azure-pipelines/pipelines.yml

```diff
@@ -374,11 +379,12 @@ stages:
 
 - stage: check
   dependsOn: ["env", "linux_x64"]
-  condition: or(eq(variables['PostSubmit'], true), and(ne(dependencies.env.repo.outputs['changed.mobileOnly'], 'true'), ne(dependencies.env.repo.outputs['changed.docsOnly'], 'true')))
   jobs:
   - job: bazel
     displayName: "linux_x64"
     dependsOn: []
+    # Skip checks if only mobile/ or docs/ have changed.
+    condition: and(not(canceled()), succeeded(), ne(stageDependencies.env.repo.outputs['changed.mobileOnly'], 'true'), ne(stageDependencies.env.repo.outputs['changed.docsOnly'], 'true'))
```

Comment:
this doesnt currently check for `PostSubmit` but im thinking that conditions are already a bit too complicated so probably would be better to hack where `mobileOnly` etc is set and make that conditional on not `PostSubmit`

Reply:
i have updated the `env` stage